### DISCUSSION TITLE
ASUS names for CPU/APU tunables

### DISCUSSION THREAD

**Discussion by flukejones** [seerge/g-helper#714] [General]
@seerge mate, ASUS provided info to me on what the tunables were actually called in ACPI, and on which CPU/GPU combo they were available. the result is this kernel patch https://lore.kernel.org/all/20230630053552.976579-9-luke@ljones.dev/

Hope this is of some use to you.

**Comment by seerge** [comment]
@flukejones hello, thanks for an update. Well it kind of confirms what is known already - you can't read values, only write. And there is some hard-limit in BIOS that will ignore values after certain amount :)
